run/GCHP/ExtData.rc.templates/ExtData.rc.carbon

```diff
@@ -432,6 +432,9 @@ CONUS_MASK_MIRROR 1 N Y - none none MASK ./HcoDir/MASKS/v2024-04/CONUS_Mask_Mirr
 
 # Comment these lines if you are not using any of the CEDS inventories
 CHINA_MASK        1 N Y - none none MASK ./HcoDir/MASKS/v2018-09/China_mask.generic.1x1.nc
+
+# For CH4 reservoir emissions from ResME
+SEASONAL_RES_MASK 1 N Y F%y2-%m2-01T00:00:00 - none none Mask ./HcoDir/CH4/v2024-01/ResME/reservoir_mask.01x01.nc
```

Comment:
Change 'N' to 'Y' (climatology column) and 'F%y2-%m2' to 'F2022-%m2' (year of climatology in file).

Reply:
Thanks for catching that. I meant to change the year to 2022 but did not realize about the climatology column. This should now be fixed in https://github.com/geoschem/geos-chem/pull/2473/commits/f25fb1c695369a7813252001a076fd66127ebc6c.

Reply:
Yeah, the climatology column is technically not necessary but it skips some logic in the model so is more efficient.